contracts/src/SP1ICS07Tendermint.sol

```diff
@@ -421,33 +433,41 @@ contract SP1ICS07Tendermint is
     }
 
     /// @notice Validates the Env public values.
-    /// @param env The public values.
-    function validateEnv(Env memory env) private view {
-        if (env.now > block.timestamp) {
-            revert ProofIsInTheFuture(block.timestamp, env.now);
+    /// @param publicClientState The public client state.
+    /// @param time The time.
+    function validateEnv(ClientState memory publicClientState, uint64 time) private view {
```

Comment:
A rename might be in order. validateEnv feels a bit weird here now.

Reply:
Renamed. Feel free to change the name in the future if you don't like the new name.